exporter/trace/cloudtrace_test.go

```diff
@@ -122,6 +123,45 @@ func TestExporter_ExportSpans(t *testing.T) {
 	assert.EqualValues(t, 1, mockTrace.len())
 }
 
+func TestExporter_DisplayNameFormatter(t *testing.T) {
+	// Initial test precondition
+	mockTrace.spansUploaded = nil
+	mockTrace.delay = 0
+	
+	spanName := "span1234"
+	format := func(s *export.SpanData) string {
+		return "TEST_FORMAT" + s.Name
+	}
+
+	// Create Google Cloud Trace Exporter
+	exp, err := texporter.NewExporter(
+		texporter.WithProjectID("PROJECT_ID_NOT_REAL"),
+		texporter.WithTraceClientOptions(clientOpt),
+		// uncomment when exporter is using bundler
+		// texporter.WithBundleCountThreshold(1),
+		texporter.WithDisplayNameFormatter(format),
+	)
+	assert.NoError(t, err)
+
+	tp, err := sdktrace.NewProvider(
+		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
+		sdktrace.WithBatcher(exp, // add following two options to ensure flush
+			sdktrace.WithScheduleDelayMillis(1),
+			sdktrace.WithMaxExportBatchSize(1),
+		))
+	assert.NoError(t, err)
+
+	global.SetTraceProvider(tp)
+	_, span := global.TraceProvider().Tracer("test-tracer").Start(context.Background(), spanName)
+	span.End()
+	assert.True(t, span.SpanContext().IsValid())
+
+	// wait exporter to flush
+	time.Sleep(20 * time.Millisecond)
```

Comment:
Sleeping for a block amount of time in tests in never ideal. To make this signficiantly more robust, and faster, a better alternative to this is to do something like
```go
assert.Eventually(t, function() {
    ...
}, 500 * time.Millisecond, 2 * time.Millisecond)
```

I wouldn't bother changing this now though. After you add the `NewPipeline` method that returns a flush function (see the Jaeger exporter), you can avoid the sleep here entirely.

Reply:
ok, sounds good. I'll leave this for now, and when I create the `NewPipeline` method, I plan to update this test and others to use the pipeline and flush function as appropriate.